--- /var/lib/kolla/venv/lib/python3.9/site-packages/neutron/conf/plugins/ml2/drivers/ovs_conf.py.ori    2023-03-16 20:22:25.760420133 +0000
+++ /var/lib/kolla/venv/lib/python3.9/site-packages/neutron/conf/plugins/ml2/drivers/ovs_conf.py        2023-03-16 20:22:54.172274148 +0000
@@ -14,6 +14,7 @@
 
 from neutron_lib import constants as n_const
 from oslo_config import cfg
+from oslo_utils import netutils
 
 from neutron._i18n import _
 from neutron.conf.agent import common
@@ -42,7 +43,7 @@
     cfg.StrOpt('tun_peer_patch_port', default='patch-int',
                help=_("Peer patch port in tunnel bridge for integration "
                       "bridge.")),
-    cfg.IPOpt('local_ip',
+    cfg.IPOpt('local_ip', default=netutils.get_my_ipv4(),
               help=_("IP address of local overlay (tunnel) network endpoint. "
                      "Use either an IPv4 or IPv6 address that resides on one "
                      "of the host network interfaces. The IP version of this "
